app/models/comment.rb

```diff
@@ -282,6 +288,7 @@ def bust_cache
 
   def synchronous_bust
     commentable.touch(:last_comment_at) if commentable.respond_to?(:last_comment_at)
+    user&.touch(:last_comment_at)
```

Comment:
Is there a scenario where the comment's user is nil?

Reply:
I had the & for the same reason as above. If I update the way described above, there will be no need here to keep it here also.

Reply:
I have removed the safe operator and updated the spec, just like the other similar case.